src/schema/artwork/__tests__/artwork.test.js

```diff
@@ -1286,7 +1286,20 @@ describe("Artwork type", () => {
       }
     `
 
+    it("is null if artwork is not enrolled in in an e-commerce program", () => {
+      artwork.acquireable = false
+
+      return runQuery(query, rootValue).then(data => {
+        expect(data).toEqual({
+          artwork: {
+            shippingInfo: null,
+          },
+        })
+      })
+    })
+
     it("is set to prompt string when its domestic_shipping_fee_cents is null and international_shipping_fee_cents is null", () => {
+      artwork.acquireable = true
```

Comment:
Super minor - but this acquirable tests can be grouped now and have one beforeAll assignment for this  acquireable=true.